packages/artifact/src/internal/shared/user-agent.ts

```diff
@@ -1,4 +1,5 @@
-var packageJson = require('../../../package.json')
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
```

Comment:
Attempted to use import for user-agent.ts, but it seems like require is our best bet if we don't want to change a bunch of config: https://stackoverflow.com/questions/50822310/how-to-import-package-json-in-typescript